segments/path.bash

```diff
@@ -2,6 +2,7 @@
 _sbp_path_color_bg=${_sbp_path_color_bg-:$_sbp_color_blue}
 _sbp_path_color_fg=${_sbp_path_color_fg-:$_sbp_color_white}
 _sbp_path_color_sep=${_sbp_path_color_sep-:$_sbp_color_grey}
+_sbp_path_disable_sep=${_sbp_path_disable_sep-:0}
```

Comment:
I like using quotes for everything that can give problems when unquoted. These just passed by me I guess. 

But the the typo @dflock found really bugged me. I even made a quickfix in helpers/colors.bash to prevent the ':' to become a part of the color codes and I just couldn't figure out where the ':' came from. Good catch, I'll get try to get that fixed now (kids are finally out of the house) :)